smrf/data/mysql_data.py

```diff
@@ -133,8 +133,9 @@ def get_data(self, table, station_ids, start_date, end_date, variables):
             self._logger.debug('Creating dataframe for {}'.format(v))
 
             # create an empty dataframe
-            dp = pd.DataFrame(index=t, columns=station_ids)
```

Comment:
This removes the index and just relies on the larger dataframe for the index? What happens if the data frame copy below doesn't have all the times? I think this was why the time index was used in the beginning, to ensure that the SMRF date range is the same as the data. At this point, I don't know if it would catch that there might be a missing row?

Reply:
Ah I see, let me monkey around with it more. The tz info was creating a totally empty df and it only worked if I removed the index. I will see where I can keep both.

Reply:
If you give me your MySQL schema dump (and some data), I am happy to fix this. I was the one breaking it in first place

Reply:
The `t` that you commented above is the index, so adding the tzinfo to that?

Reply:
@jomey  admittedly, I didn't know you could dump the schema haha. No worries though, there is no way you would have known with network specific tests like these. I think I have a fix for it.